Libs/MRML/Core/Testing/cube-utf8.mrml

```diff
@@ -1,4 +1,43 @@
-<MRML>
-<Camera
- id="vtkMRMLCameraNode1" name="camera(攝像頭)"  hideFromEditors="true"  position="1.26247 0.628757 -0.668912" focalPoint="0 0 0" viewUp="-0.566328 0.729902 -0.382773" parallelProjection="false" parallelScale="0" active="true"></Camera>
-</MRML>
+<?xml version="1.0" encoding="utf-8"?>
+<MRML>
+
+    <Camera
+        id="vtkMRMLCameraNode1"
+        name="camera(ae=æ AE=Æ a:=ä A:=Ä o|=ø O|=Ø o:=ö O:=Ö ao=å Ao=Å)"
+        hideFromEditors="true"
+        position="1.26247 0.628757 -0.668912"
+        focalPoint="0 0 0"
+        viewUp="-0.566328 0.729902 -0.382773"
+        parallelProjection="false"
+        parallelScale="0"
+        active="true">
+    </Camera>
+
+    <VolumeArchetypeStorage
+        id="vtkMRMLVolumeArchetypeStorageNode1" name=""  hideFromEditors="true"
+        fileName="TestData/fixed.nrrd" >
+    </VolumeArchetypeStorage>
+
+    <VolumeDisplay
+        id="vtkMRMLScalarVolumeDisplayNode1"
+        name="árvíztűrő tükörfúrógép"
```

Comment:
Is this what was intended?

Reply:
Yes, this is a well-known phrase that contain all the accented characters of Hungarian language. (to be precise: well-known to Hungarians)

Reply:
😄